### PR TITLE
Creating a permanent identifier for the bop ontology

### PR DESCRIPTION
**Pull request by AlexDonkers**
Dear maintainer, 

Could you please create a permanent identifier for the bop ontology?
I hope I provided you with the correct documents via this way. 

Project: Building Performance Ontology
Prefix: bop
Namespace: https://w3id.org/bop

Best, Alex
a.j.a.donkers@tue.nl

### CODE REVIEW

**Comment by dgarijo** [comment]
@AlexDonkers,
you are copying directly the htaccess by Widoco, which is not correct because we don't host the files in here.
You should host the files in your target GitHUb repository and then copy the URL in the htaccess. 
See the example I did here: https://github.com/perma-id/w3id.org/blob/master/example/.htaccess for reference.

**Comment by AlexDonkers** [comment]
@dgarijo Thank you for your comment. Excuse me, I didn't check the links. They are updated now.

**Comment by dgarijo** [comment]
@AlexDonkers,
please remove the `rewritebase` line, because the `myDocumentation` folder does not exist.

File: bop/.htaccess [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+# Rewrite engine setup
+RewriteEngine On
+#Change the path to the folder here
+RewriteBase /myDocumentation
```
Comment (dgarijo) [inline bop/.htaccess:14]: Remove line 14

Comment (AlexDonkers) [reply bop/.htaccess]: @dgarijo Deleted line 14.

**Pull request merged**